class/SyncHandlers/SyncHandlerBase.php

```diff
@@ -239,4 +243,24 @@ protected static function sync_all_paginated(string $url, int $page_number = 1,
 
     return;
   }
+
+  protected static function check_sync_of_authors($postarr)
+  {
+    if ($postarr["post_type"] === PASSLESYNC_POST_TYPE) {
+      $meta = $postarr["meta_input"];
+      $all_author_shortcodes_on_post = array_merge($meta["post_author_shortcodes"], $meta["post_coauthors_shortcodes"]);
+
+      $author_profiles = get_posts([
+        'post_type' => PASSLESYNC_AUTHOR_TYPE,
+      ]);
+
+      foreach ($all_author_shortcodes_on_post as $author_shortcode) {
+        $author_post = wp_filter_object_list($author_profiles, array('post_name' => $all_author_shortcodes_on_post));
```

Comment:
Is this variable meant to mean "$author_exists_in_wp"? A better name here may be useful